cmd/resultscollector/main.go

```diff
@@ -31,6 +31,7 @@ import (
 
 	backoff "github.com/cenkalti/backoff/v3"
 	"github.com/dsnet/compress/bzip2"
+	libgocrypto "github.com/openshift/library-go/pkg/crypto"
```

Comment:
Does this implement the FIPS validated crypto on RH platforms?

Reply:
Supposedly. Note that I didn't introduce the usage of this library, merely moved the import.